R/viz.R

```diff
@@ -4220,57 +4332,75 @@ fancy_circos_plot = function(this_sample,
     n_track = 4
   }
 
-  #get variants
-  svs = get_combined_sv(sample_ids = this_sample, projection = projection)
+  if(manta_calls){
+    #get variants
+    svs = get_combined_sv(sample_ids = this_sample, projection = projection)
 
-  #filter on vaf
-  svs = dplyr::filter(svs, VAF_tumour > vaf_cutoff)
+    #filter on vaf
+    svs = dplyr::filter(svs, VAF_tumour > vaf_cutoff)
 
-  #subset on relevant variables
-  svs_df = dplyr::select(svs, CHROM_A, START_A, END_A, CHROM_B, START_B, END_B, manta_name)
+    #subset on relevant variables
+    svs_df = dplyr::select(svs, CHROM_A, START_A, END_A, CHROM_B, START_B, END_B, manta_name)
 
-  #split manta_name variable
-  svs_df = data.frame(svs_df$CHROM_A, svs_df$START_A, svs_df$END_A, svs_df$CHROM_B, svs_df$START_B, svs_df$END_B, do.call(rbind, strsplit(svs_df$manta_name, split = ":", fixed = TRUE)))
+    #split manta_name variable
+    svs_df = data.frame(svs_df$CHROM_A, svs_df$START_A, svs_df$END_A, svs_df$CHROM_B, svs_df$START_B, svs_df$END_B, do.call(rbind, strsplit(svs_df$manta_name, split = ":", fixed = TRUE)))
 
-  #rename variables
-  names(svs_df)[1] = "CHROM_A"
-  names(svs_df)[2] = "START_A"
-  names(svs_df)[3] = "END_A"
-  names(svs_df)[4] = "CHROM_B"
-  names(svs_df)[5] = "START_B"
-  names(svs_df)[6] = "END_B"
-  names(svs_df)[7] = "TYPE"
+    #rename variables
+    names(svs_df)[1] = "CHROM_A"
```

Comment:
How many columns are there in total and what are the original column names? I think I might be missing the step `#split manta_name variable` above. Maybe there is no column renaming required if you use `dplyr::select()`?

If renaming all you can just call this
```
colnames(svs_df) = c("CHROM_A",...,"TYPE")
```
or if there are many columns, this can become
```
colnames(svs_df)[1:7] = c("CHROM_A",...,"TYPE")
```